server/sonar-server/src/main/java/org/sonar/server/es/RecoveryIndexer.java

```diff
@@ -68,17 +70,22 @@
   private final UserIndexer userIndexer;
   private final RuleIndexer ruleIndexer;
   private final ActiveRuleIndexer activeRuleIndexer;
+  private final PermissionIndexer permissionIndexer;
+  private final IssueIndexer issueIndexer;
   private final long minAgeInMs;
   private final long loopLimit;
 
   public RecoveryIndexer(System2 system2, Settings settings, DbClient dbClient,
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Constructor has 8 parameters, which is greater than 7 authorized. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS00107)